test/discover.test.js

```diff
@@ -136,6 +136,72 @@ describe('Discover model primary keys', function() {
       }
     });
   });
+
+  it('should return just the primary keys and not the foreign keys for inventory', function(done) {
```

Comment:
`it('returns only primary keys', function(done) {`

Leave the database specifics out of the test name